compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -1115,15 +1115,11 @@ class Namer { typer: Typer =>
             WildcardType
       } getOrElse WildcardType
 
-      // println(s"final inherited for $sym: ${inherited.toString}") !!!
-      // println(s"owner = ${sym.owner}, decls = ${sym.owner.info.decls.show}")
-      def isInline = sym.is(FinalOrInlineOrTransparent, butNot = Method | Mutable)
-
       // Widen rhs type and eliminate `|' but keep ConstantTypes if
       // definition is inline (i.e. final in Scala2) and keep module singleton types
```

Comment:
I think that this comment needs to be updated.

Reply:
Good point